pykechain/client.py

```diff
@@ -1105,6 +1105,106 @@ def create_activity(
             parent._cached_children.append(new_activity)
         return new_activity
 
+    def clone_activities(
+        self,
+        activities: List[Union[Activity2, Text]],
+        parent: Union[Activity2, Text],
+        activity_update_dicts: Optional[Dict] = None,
```

Comment:
do you do this in other places as well. To have a dict of `id: {field_values}`? Backend does a list of dicts. Bit different, endresult the same. Do you have a specific reason why you have this? Is this better suited for handling in code?

Reply:
To separate the clone from the update. I already had difficulties interpreting the API endpoint because of this list of dicts, so this separation makes it simpler if you just want to clone some stuff.

Reply:
Small addition: I perceive Pykechain (with its checking on inputs) to act as _convenience_ for any developer working with the API: Very often it is hard to interpret the raw API and its responses/errors with little context. This drives my decisions in these cases.